ui/src/container-list.tsx

```diff
@@ -16,12 +16,21 @@ import {Link} from "@mui/material";
 import {v1} from "@docker/extension-api-client-types";
 import DeleteIcon from '@mui/icons-material/Delete';
 import DownloadIcon from '@mui/icons-material/Download';
+import { GithubUser, Package, PackageVersion } from './types';
 
-function Row(props: { setState: any, onChange: any, user: any, pkg: any, client: v1.DockerDesktopClient }) {
```

Comment:
The type `any` may often lead to unintended bug.
(ex. property access for nullable object, type unmatching, etc...)